keras_cv/layers/preprocessing/mosaic.py

```diff
@@ -269,18 +266,6 @@ def _batch_augment(self, inputs):
         self._validate_inputs(inputs)
         return super()._batch_augment(inputs)
 
-    def call(self, inputs, training=True):
-        if training is True:
-            _, metadata = self._format_inputs(inputs)
-            if metadata[BATCHED] is not True:
```

Comment:
here we do have a mild regression.  Mosaic DOES need to check whether or not inputs are batched, so lets kepe this check in `call()`.

Reply:
Done.